ui/src/ducks/app/nodes.js

```diff
@@ -434,104 +361,60 @@ export function* deployNode({ payload }) {
     yield put(
       addNotificationErrorAction({
         title: intl.translate('node_deployment'),
-        message: result.error
-      })
+        message: result.error,
+      }),
     );
   } else {
-    yield call(subscribeDeployEvents, { jid: result.return[0].jid });
-    addJobLocalStorage(result.return[0].jid, payload.name);
-    yield call(fetchNodes);
-  }
-}
-
-export function subSSE(eventSrc) {
-  const subs = emitter => {
-    eventSrc.onmessage = msg => {
-      emitter(msg);
-    };
-    eventSrc.onerror = () => {
-      emitter(END);
-    };
-    return () => {
-      eventSrc.close();
-    };
-  };
-  return eventChannel(subs);
-}
-
-export function* sseSagas({ payload }) {
-  const eventSrc = new EventSource(
-    `${payload.url}/events?token=${payload.token}`
-  );
-  const channel = yield call(subSSE, eventSrc);
-  while (true) {
-    const msg = yield take(channel);
-    const data = JSON.parse(msg.data);
-    const jobs = yield select(state => state.app.salt.jobs);
-
-    yield all(
-      jobs.map(jid => {
-        if (data.tag.includes(jid)) {
-          return call(updateDeployEvents, jid, data);
-        }
-        return data;
-      })
+    yield put(
+      addJobAction({
+        name: `deploy-node/${payload.name}`,
+        jid: result.return[0].jid,
```

Comment:
I actually didn't want to add too much in `localStorage`. But since we don't care too much about "dimensions" when it comes to JSON, I think it actually makes a lot of sense to switch to `type` + some arbitrary number of attributes for consumers to read from, in place of some known format to encode these attributes in a single `name` string.

Good point :)